src/scripts/main.js

```diff
@@ -1,3 +1,47 @@
 'use strict';
 
 // write code here
+
+const table = document.querySelector('.field > tbody');
+const buttons = document.querySelector('.container');
+
+function modifyColumns(rezising) {
+  switch (rezising) {
+    case 'append-row':
+      table.appendChild(table.querySelector('tr').cloneNode(true));
+      break;
+    case 'remove-row':
+      table.deleteRow(table.rows.length - 1);
+      break;
+    case 'append-column':
+      [...table.rows].forEach((row) => {
+        row.appendChild(row.cells[row.cells.length - 1].cloneNode(true));
+      });
+      break;
+    case 'remove-column':
+      [...table.rows].forEach((row) => {
+        row.removeChild(row.cells[row.cells.length - 1]);
```

Comment:
Check if the row has any cells before attempting to remove one. If `row.cells.length` is zero, this will throw an error.